tests/test_parser.py

```diff
@@ -21,44 +21,79 @@
 
 
 class TestsGetPage(unittest.TestCase):
-    def setUp(self):
-        pass
 
-    def tearDown(self):
-        pass
-
-    @patch('requests.get')
-    def test_perform_get_request_uses_provided_url_for_request(
-            self,
-            mock_request):
-        mock_request.return_value.status_code = 200
+    def test_perform_get_request_uses_provided_url_for_request(self):
+        session = Mock(
```

Comment:
you mock the session in every single test. We might want to create new class for get requests testing and patch it as a decorator with some predefined values and if needed adjust needed properties/methods/etc in the specific test

Reply:
As  I see that the same is done with post function tests, then let's create separate issue to fix that

Reply:
https://github.com/dkultasev/smarsy-py/issues/8 issue is created for that